src/jarowinkler/CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ function(create_cython_target _name)
 endfunction(create_cython_target)
 
 create_cython_target(_initialize_cpp)
-Python_add_library(_initialize_cpp MODULE ${_initialize_cpp})
+Python_add_library(_initialize_cpp MODULE WITH_SOABI ${_initialize_cpp})
```

Comment:
Only PyPy seems to care if this is missing. Maybe to avoid reading a CPython .so?